### ISSUE TITLE
vita keeps turning off during the epilepsy warning message

### ISSUE THREAD

**Issue by umbrellaman11** [dots-tb/LOLIcon#18]
@dots-tb 

my vita keeps turning off right before the epilepsy warning. i have the latest version of your plugin but this keeps happening whenever i turn my vita on/off, its random but today its happened 5 times and am getting fed up of the "vita did not shutdown properly" message when in the live area after i successfully boot back into the vita.

i did save both config files default and main for the live area. unsure as to why this is affecting even after the hotfix update.

yes my plugin text is on the first line on config.txt under kernel.

still no luck though as this happens and needs to be addressed.

**Comment by dots-tb** [comment]
Before it crashes, do you see the green text saying that it loaded the configuration?

**Comment by umbrellaman11** [comment]
yes, it says configuration loaded : 4 then stalls the vita from loading right as the  epilepsy warning message appears and automatically shuts down the vita, the blue light blinks about 3-5 times, and when I turn the vita back on a few times when it does not do this, I get a warning message on live area that the vita encountered an error the last time it was turned off [okay]/[report this problem].

I turned off all my other plugins, and ran this again, same thing happens, this is definitely the lolicon plugin thats doing it, as when i run the vita without this plugin activated everything works fine again.

**Comment by ghost** [comment]
For my case, placing LOLIcon just a line below *KERNEL and disabling the UDCD_UVC plugin while LOLIcon is active solved this problem.

**Comment by umbrellaman11** [comment]
> placing LOLIcon just a line below *KERNEL

that was already done, and i dont have the udcd_uvc plugin at all on my vita. :(

**Comment by JosePepowner** [comment]
This isn't a bug. I know its a year later and maybe you solved it by now but this is mainly for future viewers. DO NOT set your main or default overclock to minimum. The system UI won't be able to load at such low frequency so thats why it crashes. 

To restore just factory reset your vita in the Recovery mode. Or be smart and do what I did because this shit happened to me. On your memory card keep your molecule bubble installed so you can edit the config file if you need to append the lolicon plugin. Remember you can always boot the vita without plugins but still hacked so this allows you to just remove the plugin from launching then you can edit lolicons config files and remove the frequency or just delete the config file all together. Then boot like normal.
